pkg/api/dtos/models.go

```diff
@@ -73,12 +73,7 @@ func GetGravatarUrlWithDefault(text string, defaultText string) string {
 		return GetGravatarUrl(text)
 	}
 
-	reg, err := regexp.Compile("[^a-zA-Z0-9]+")
-
-	if err != nil {
-		return ""
-	}
-
+	reg := regexp.MustCompile("[^a-zA-Z0-9]+")
```

Comment:
Put into package init, that way a panic it is caught at package init time. 

and also only need to compile once.